packages/client/serviceWorker/sw.ts

```diff
@@ -31,7 +31,7 @@ const onInstall = async (_event: ExtendableEvent) => {
   // if this is their first service worker, fetch it all
   if (!oldStaticCacheName) {
     console.log('Installing service worker')
-    return newCache.addAll(urls)
+    return newCache.addAll(urls).catch(console.error)
```

Comment:
`fetchAll` fetches all URLs, even if some fail, but rejects the final promise in that case, see https://w3c.github.io/ServiceWorker/#cache-addAll